openadapt/scrub.py

```diff
@@ -0,0 +1,180 @@
+"""Script to scrub text of all PII/PHI.
+
+Usage:
+
+    $ python openadapt/scrub.py scrub_text str_arg
+    $ python openadapt/scrub.py scrub_image PIL.Image_arg
+    $ python openadapt/scrub.py scrub_dict dict_arg
+    $ python openadapt/scrub.py scrub_list_dicts list_of_dicts_arg
+
+"""
+
+from PIL import Image
+from presidio_anonymizer.entities import OperatorConfig
+import fire
+
+from openadapt import config, utils
+
+
+def scrub_text(text: str, is_hyphenated: bool = False) -> str:
+    """
+    Scrub the text of all PII/PHI using Presidio ANALYZER.TRF and Anonymizer
+
+    Args:
+        text (str): Text to be scrubbed
+
+    Returns:
+        str: Scrubbed text
+    """
+
+    if text is None:
+        return None
+
+    if is_hyphenated and not (
+        text.startswith("<") or text.endswith(">")
+    ):
+        text = "".join(text.split("-"))
```

Comment:
Please re-use the constants in https://github.com/MLDSAI/OpenAdapt/blob/main/openadapt/models.py#L204

It may be time to move these into config.py